src/Main.java

```diff
@@ -1,5 +1,33 @@
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println(
+                "Билет 2\nЗадача: напишите программу, которая проверяет, " +
+                        "является ли строка палиндромом.\n");
```

Comment:
Scanner scanner = new Scanner(System.in);
 task1(scanner);
        task2(scanner);